<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="96783" started-at="2020-03-05T03:42:34Z" finished-at="2020-03-05T03:44:11Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="96783" started-at="2020-03-05T03:42:34Z" finished-at="2020-03-05T03:44:11Z">
      <class name="com.checkingLinks.test.CheckLinksTest">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:com.checkingLinks.test.CheckLinksTest@51081592]" name="beforeClass" is-config="true" duration-ms="13982" started-at="2020-03-04T21:42:34Z" finished-at="2020-03-04T21:42:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="checkingLinks()[pri:0, instance:com.checkingLinks.test.CheckLinksTest@51081592]" name="checkingLinks" duration-ms="82771" started-at="2020-03-04T21:42:48Z" finished-at="2020-03-04T21:44:11Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[There are broken links expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: There are broken links expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.checkingLinks.test.CheckLinksTest.checkingLinks(CheckLinksTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkingLinks -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:com.checkingLinks.test.CheckLinksTest@51081592]" name="afterClass" is-config="true" duration-ms="0" started-at="2020-03-04T21:44:11Z" finished-at="2020-03-04T21:44:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.checkingLinks.test.CheckLinksTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
